lib/Doctrine/Pager/Layout.php

```diff
@@ -305,7 +305,6 @@ public function removeMaskReplacement($oldMask)
      */
     public function cleanMaskReplacements()
     {
-        $this->_maskReplacements = null;
         $this->_maskReplacements = array();
```

Comment:
This is defined as an empty array for the default value in the initial definition, matching that (so don't have to mark the property as nullable).  There aren't any null checks on this property in this class or anything that might extend it (protected visibility).